policy/modules/kernel/files.if

```diff
@@ -2909,6 +2909,24 @@ interface(`files_mounton_etc_dirs',`
 	allow $1 etc_t:dir mounton;
 ')
 
+########################################
+## <summary>
+##	Watch /etc symlinks
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`files_watch_etc_symlinks', `
+	gen_require(`
+		type etc_t;
+	')
+
+	allow $1 etc_t:lnk_file watch;
+')
+
```

Comment:
This should go down with the other `etc_t` symlink interfaces, around line 3220.